CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### Enhancements
 
 - Can now create control flow graphs to visualize the execution of your program.
+- The `TopLevelCodeChecker` can now check for the proper naming style usage for type alias assignment statements.
```

Comment:
This is a good start but a bit misleading as written. Really the point of this check is that `TopLevelCodeChecker` now *allows* type alias assignment statements at the top level, i.e., *doesn't* flag such lines of code as an error